docs/features/REST-sphubsite-type.md

```diff
@@ -0,0 +1,24 @@
+---
+title: SPHubSite REST type
+description: SPHubSite REST type
+ms.date: 2/26/2018
+---
+
+# SPHubSite REST type
+
+> [!IMPORTANT]
+> The hub sites feature is currently in preview and is subject to change. It is not currently supported for use in production environments.
+
+Contains data describing a SharePoint Hub site.
+
+
+|Name    |Type   |Description |
+|--------|-------|------------|
+|ID      |GUID   |Identifies the hub site. |
+|Title   |string |The display name of the hub. |
```

Comment:
Table has several instances of "hub" or "site hub" rather than "hub site"

Reply:
fixed